release/models/network-instance/openconfig-route-summary.yang

```diff
@@ -0,0 +1,96 @@
+module openconfig-route-summary {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/route-summary";
+
+  prefix "oc-rtsummary";
+
+  import openconfig-extensions { prefix "oc-ext"; }
+  import openconfig-yang-types { prefix "oc-yang-types"; }
+  import openconfig-policy-types { prefix "oc-pol-types"; }
+  import openconfig-network-instance { prefix "oc-ni"; }
+
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This module provides summary of route counts per route type for each network
+    instance.";
+
+  oc-ext:openconfig-version "1.0.0";
+
+  revision "2022-12-21" {
+    description
+      "Initial version.";
+    reference "1.0.0";
+  }
+
+  grouping route-counter {
+    leaf route-type {
+      type identityref {
+        base "oc-pol-types:INSTALL_PROTOCOL_TYPE";
+      }
+      description
+        "Route type that keys the route count list.";
+    }
+    leaf count {
+      type oc-yang-types:counter64;
+      description
+        "Count for the route type.";
```

Comment:
Can we add more clarification around what this count indicates? Is it the number received and in the RIB from the protocol, but not necessarily installed, or is it installed routes?

Reply:
We decided to break `count` into two counters: `total-count` for all routes in the AFT,  `installed-count` for routes installed in the hardware. The tree output has been updated.

Reply:
@robshakir The comments from you last time are all dealt with. Can you please take another look?